ktor-client/ktor-client-winhttp/windows/interop/winhttp.def

```diff
@@ -0,0 +1,142 @@
+package = platform.windows
```

Comment:
is it ok to use `platform` package here?

Reply:
Naturally WinHTTP is a part of Windows platform library. But kotlin native team cautiously adds new parts in it: https://youtrack.jetbrains.com/issue/KT-39747

Reply:
The problem will arise, if they decide to add it someday
I would prefer to use some other package to prevent future issues in such case

Reply:
Hi. I suggest something like this: https://gist.github.com/sbogolepov/276ce9218e6939165362af251fe4301c. The exact package name is not important, but it shouldn't be `platform.windows`.
1. `platform.windows` is enormous which is bad for many things, including IDE experience. We can't split it easily because of backward API&ABI compatibility, but at least we can stop adding new declarations to it.
2. `winhttp.h` and `wininet.h` have constants with the same name but different value (`INTERNET_SCHEME_HTTP `). Putting them into one package will result into shadowing of one of them.  
3. `GetHResultFromError` and `MakeLanguageId` can be implemented in Kotlin, thus keeping interop package clean.
My suggestion is very close to something that I'm going to implement in 1.8.0. So after its release, it will be possible to switch to `platform.winhttp` without major changes.